query/templates/search/v710/countQuery.tmpl

```diff
@@ -19,11 +19,11 @@
    },
   "size": 0,
   "aggregations": {
-  		"docCounts": {
```

Comment:
(question) I am assuming that the content type counts were referred as `docCounts` before? Or is `docCounts` something else?